.eslintrc

```diff
@@ -20,11 +21,1548 @@
         "worker": true
     },
     "rules": {
-        // Eslint:recommended https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js (all errors set to warning)
+        // enforces getter/setter pairs in objects
```

Comment:
Ez a sok rule az Airbnb-től jön, ugye? Nem lehetne azokat egy külön fájlban tartani? Ők mit javasolnak rá, hogy ne copy-paste-elni kelljen tőlük?

A másik, hogy mire vannak pontosan a TODO commentek? Mert olyanokat nem kellene benne hagyni.

Reply:
A TODO kommentek is copy-pastek az airbnb-től.

A külön fileos egy olyan kérdés, hogy lehet, de akkor egy egész mappát kéne másolni gyökérbe

Reply:
Az nem probléma. Submodule-ként is hozzáadhatod esetleg az ő repojukat, nem?

Reply:
Tehát van valamilyen best practice arra, hogy 

1. A rule-okat amiket mi akarunk használni, hogy lehet szebben bekapcsolni azon kívűl hogy copy paste?
2. a rule-on belül ha mi valamiben eltérünk, akkor akár egy ettől külön file-ben hogy lehet csak az override-okat (pl a 120-as sorhosszúság) tartani.

Reply:
Az ő repojukat kicsit overkill lenne szerintem berakni, mert nem csak az eslint van benne, viszont úgy megcsinálni, hogy ami ki lett copy pastelve onnan van egy base fileban, és külön amit mi írunk override arra egy másikban

Reply:
A base file érintetlenül bemásolása és a mi cuccaink csak egy override fájlban is teljesen jó. Viszont előtte azért nézzük meg a submodule-t, mert az nem egy repo: https://github.com/airbnb/javascript De tényleg nincs semmi egyéb tippjük erre?

Reply:
Egyéb tippet az airbnb-nél nem találtam, csak hogy forkoljuk, viszont megcsináltam a base fileos megoldást

Reply:
OK, akkor azt mindenképp dokumentáld kérlek, hogy az a fájl honnan származik és hogyan frissíthető, hogy ha fél év múlva belenyúlunk, ne kelljen gondolkozni.

Reply:
Ennek dokumentálása kimaradt.

Reply:
Ez benne van a readme-ben, 75-76 sor

Reply:
Mármint a "fájl [az Airbnb-é] honnan származik és hogyan frissíthető" maradt ki. Tehát, ha most nyitom meg ezt a projectet először, akkor tudjam ezt is, mert most az nem derül ki, hogy honnan töltötted le, meg így az se, hogy honnan kellene frissíteni. Amúgy akkor már azt is linkeld be, ahol leírják a best practice-eiket, mert az is hasznos.

Reply:
Oké oké így már értem köszi

Reply:
A "hogyan frissíthető" még mindig nincs benne :).